packages/cli/src/commands/exec.js

```diff
@@ -0,0 +1,94 @@
+import path from 'path'
+
+import babelRequireHook from '@babel/register'
+import Listr from 'listr'
+import VerboseRenderer from 'listr-verbose-renderer'
+import terminalLink from 'terminal-link'
+
+import { getPaths } from 'src/lib'
+import c from 'src/lib/colors'
+import { generatePrismaClient } from 'src/lib/generatePrismaClient'
+
+const runScript = async (scriptPath, scriptArgs) => {
+  // Import babel config for running script
+  babelRequireHook({
+    extends: path.join(getPaths().api.base, '.babelrc.js'),
+    extensions: ['.js', '.ts'],
+    plugins: [
+      [
+        'babel-plugin-module-resolver',
+        {
+          alias: {
+            api: getPaths().api.base,
+          },
```

Comment:
I would love to use `$` to denote a portal to another package.
```suggestion
          alias: {
            '$api': getPaths().api.base,
          },
```